src/js/hca/containers/ConfirmationPage.jsx

```diff
@@ -22,33 +21,38 @@ export class ConfirmationPage extends React.Component {
   }
 
   render() {
-    const time = _.get('form.submission.response.timestamp', this.props);
+    const { submission, data } = this.props.form;
+    const { response } = submission;
+    const name = data.veteranFullName;
 
     return (
       <div>
-        <div className="usa-width-one-sixth medium-2 columns">
-          <i className="fa fa-check-circle hca-success-icon"></i>
-        </div>
-        <div className="usa-width-five-sixths medium-10 columns">
-          <h4 className="success-copy">You have successfully submitted your application for health care.</h4>
+        <h3 className="confirmation-page-title">Claim received</h3>
+        <p>Normally processed within <strong>a week</strong>.</p>
+        <p>
+          We may contact you for more information or documents.<br/>
+          <i>Please print this page for your records.</i>
+        </p>
+        <div className="inset">
+          <h4>Health Care Benefit Claim <span className="additional">(Form 10-10EZ)</span></h4>
+          <span>for {name.first} {name.middle} {name.last} {name.suffix}</span>
+
+          {response && (response.timestamp || response.formSubmissionId) && <ul className="claim-list">
```

Comment:
I think you can assume that if you have a response, you have a timestamp and a submission id.

Reply:
Thanks Jeff, updated here and in the test.